release-notes.txt

```diff
@@ -1,3 +1,688 @@
++     assert(options_tested);
+# endif
+-     netif->cur_hl = GNRC_NETIF_DEFAULT_HL;
+- #ifdef MODULE_GNRC_IPV6_NIB
+-     gnrc_ipv6_nib_init_iface(netif);
+- #endif
+-     if (netif->ops->init) {
+-         netif->ops->init(netif);
+-     }
+
+/* split the semantic of gnrc_netif_is_6ln() into two check functions */
+/* is the interface performing 6Lo-ND as specified in RFC 6775. */
+- bool gnrc_netif_is_6ln(const gnrc_netif_t *netif);
++ bool gnrc_netif_is_6lo(const gnrc_netif_t *netif);
++ static inline bool gnrc_netif_is_6ln(const gnrc_netif_t *netif
+
+/* add function to start a shell and exit once EOF is reached. */
++ void shell_run_once(const shell_command_t *commands, char *line_buf, int len);
+
+/* return  reassembly buffer entry created or updated */
+- void gnrc_sixlowpan_frag_rb_add(gnrc_netif_hdr_t *netif_hdr,
+-                                 gnrc_pktsnip_t *frag, size_t offset,
+-                                 unsigned page);
++ gnrc_sixlowpan_frag_rb_t *gnrc_sixlowpan_frag_rb_add(gnrc_netif_hdr_t *netif_hdr,
++                                                      gnrc_pktsnip_t *frag,
++                                                      size_t offset, unsigned page);
+
+/* move gnrc_sixlowpan_frag_rb_dispatch_when_complete() out of
+   gnrc_sixlowpan_frag_rb_add(). The caller of the latter function is now
+   responsible to call it. */
++ int gnrc_sixlowpan_frag_rb_dispatch_when_complete(gnrc_sixlowpan_frag_rb_t *rbuf,
++                                                   gnrc_netif_hdr_t *netif);
+
+/* don't assume interface specifier is an int for splitting, add functions to
+   split int and string */
+-  * @return          interface number or -1 if none specified
++  * @return          string containing the interface specifier.
++  * @return          NULL if no interface was specified.
++  */
+- static inline int ipv6_addr_split_iface(char *addr_str)
++ static inline char *ipv6_addr_split_iface(char *addr_str)
+
+- int ipv6_addr_split(char *addr_str, char seperator, int _default);
++ int ipv6_addr_split_int(char *addr_str, char seperator, int _default);
++ char *ipv6_addr_split_str(char *addr_str, char separator);
+
+/* changed the address getter and setter functions to avoid byte order confusion. */
+- uint16_t at86rf2xx_get_addr_short(const at86rf2xx_t *dev);
++ void at86rf2xx_get_addr_short(const at86rf2xx_t *dev, network_uint16_t *addr);
+
+- void at86rf2xx_set_addr_short(at86rf2xx_t *dev, uint16_t addr);
++ void at86rf2xx_set_addr_short(at86rf2xx_t *dev, const network_uint16_t *addr);
+
+- uint64_t at86rf2xx_get_addr_long(const at86rf2xx_t *dev);
++ void at86rf2xx_get_addr_long(const at86rf2xx_t *dev, eui64_t *addr);
+
+- void at86rf2xx_set_addr_long(at86rf2xx_t *dev, uint64_t addr);
++ void at86rf2xx_set_addr_long(at86rf2xx_t *dev, const eui64_t *addr);
+```
+
+Deprecations
+========================
+
+Warnings
+--------
+
+- GNU make <4 is deprecated in RIOT
+- gnrc_nettest: will be removed after 2020.07 release
```

Comment:
```suggestion
- gnrc_nettest: will be removed after release 2020.07
```

Reply:
Maybe move this at the end of the list to keep a chronological order (closest from removal at the top)